docs/_sass/layout.scss

```diff
@@ -469,35 +469,53 @@ hr {
   display: grid;
   grid-template-columns: 1fr 1fr 1fr;
 }
+.custom-desc-container{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 5px;
+  align-items: center;
+  justify-items: center;
+}
+
 .custom-desc-div {
   display: flex;
   padding: 0rem 0;
+  height: 100%;
+  width: 100%;
 }
 
-@media only screen and (max-width: 992px) {
-  .custom-desc-div {
-    display: flex;
-    flex-direction: column;
-    padding: 0rem;
-  }
-}
+// @media only screen and (max-width: 992px) {
```

Comment:
there's no need to comment out code if we're not using it can be used.

Reply:
So should I remove the code instead of uncommenting it?

Reply:
In this case, yes, we're good to go in terms of deleting this unused CSS.

Reply:
Alright perfect. I will make necessary adjustments